lib/db.js

```diff
@@ -573,7 +573,6 @@ var createCollection = function(self, name, options, callback) {
  * @param {boolean} [options.serializeFunctions=false] Serialize functions on any object.
  * @param {boolean} [options.strict=false] Returns an error if the collection does not exist
  * @param {boolean} [options.capped=false] Create a capped collection.
- * @param {boolean} [options.autoIndexId=true] Create an index on the _id field of the document, True by default on MongoDB 2.2 or higher off for version < 2.2.
```

Comment:
hm, we actually say for >= 2.2 it is `true` by default, and `false` for < 2.2. I think we still need to keep this in, and document it as being deprecated at this point.